src/modules/awake/Awake/Core/APIHelper.cs

```diff
@@ -83,6 +84,8 @@ private static bool SetAwakeState(EXECUTION_STATE state)
 
         public static void SetIndefiniteKeepAwake(Action<bool> callback, Action failureCallback, bool keepDisplayOn = false)
         {
+            PowerToysTelemetry.Log.WriteEvent(new Awake.Telemetry.AwakeTimedKeepAwakeEvent());
```

Comment:
Should `AwakeIndefinitelyKeepAwakeEvent` be here?

Reply:
Good spotting ;) Thank you! I've just fixed it.